detox/android/detox/src/full/java/com/wix/detox/reactnative/idlingresources/network/NetworkingModuleReflected.kt

```diff
@@ -3,20 +3,28 @@ package com.wix.detox.reactnative.idlingresources.network
 import android.util.Log
 import com.facebook.react.bridge.ReactContext
 import com.facebook.react.modules.network.NetworkingModule
+import com.wix.detox.reactnative.ReactNativeInfo
 import okhttp3.OkHttpClient
 import org.joor.Reflect
 import org.joor.ReflectException
 
 
 private const val LOG_TAG = "RNNetworkingModuleRefl"
 
-private const val FIELD_OKHTTP_CLIENT = "mClient"
+private const val FIELD_OKHTTP_CLIENT_RN79 = "mClient"
```

Comment:
Ambiguous and confusing. How does this sound?
```suggestion
private const val FIELD_OKHTTP_CLIENT_PRE_RN80 = "mClient"
```